react-web/src/components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import './ConnectWallet.scss';
+import Modal from "components/Modal/Modal";
+import Button from "components/Button/Button";
+
+import { useAppDispatch } from "store/store";
+import { getProfileDetails } from "store/slices/auth.slice";
+import Toast from "components/Toast/Toast";
+
+
+const wallets: Array<string> = ['lace', 'nami', 'yoroi']
+
+declare global {
+    interface Window {
+        cardano:any;
+    }
+}
+let CardanoNS = window.cardano;
+
+const ConnectWallet = () => {
+    const dispatch = useAppDispatch();
+    const [wallet, setWallet] = useState(null)
+    const [address, setAddress] = useState(null)
+
+    const loadWallet = async (walletName: string) => {
+        try {
+            const enabledWallet = await CardanoNS[walletName].enable();
+            setWallet(enabledWallet)
+            if (enabledWallet) {
+                setAddress(await enabledWallet.getChangeAddress())
+            }
+        } catch (err) {
+            // do nothing
+            console.log(err);
+            <Toast message={""}/>
```

Comment:
It is supposed to show a default error message (for the time-being as a placeholder) in case connecting to the wallet/enable the wallet fails

Reply:
But this being a callback ( even in the future ) any component you would add there will have no effect since it is not a part of your DOM tree. am I right?

Reply:
True, need to recheck. will remove for now